### ISSUE TITLE
Drugs should be an array, not a string

### ISSUE THREAD

**Issue by dsifford** [aliemteam/aliemcards#102]
Drugs currently exist like this in the frontmatter.

```yml
[...]
updates: 2017/02/14
cateories:
    - Toxicology
drugs: amantadine, buproprion, chlorpromazine, desipramine, fluoxetine, labetalol, Methylphenidate, Phentermine  # ...etc
```
This creates extra work. 

It should be like this

```yml
[...]
updates: 2017/02/14
cateories:
    - Toxicology
drugs: [ amantadine, buproprion, chlorpromazine, desipramine, fluoxetine, labetalol, Methylphenidate, Phentermine ]
```

**Comment by michellealiem** [comment]
OK. While we talking about this, should the names be all lower case too?

**Comment by dsifford** [comment]
@michellealiem Yep. Good catch.

**Comment by jvoros** [comment]
Is this even useful? We're doing a lot of hand wringing over something I
set up on a lark. Not sure we need the feature now. If all drugs in <span>
then programatically adding later is possible.

**Comment by dsifford** [comment]
@jvoros Not sure I understand the last part of what you're saying.

**Comment by jvoros** [comment]
@dsifford during build phase, not hard to parse markdown files, pull out
drug names wrapped by <span class="drug"><span> and create the per card
drug array that way. In fact, that's how it should work now anyway, now
that I type that out.

Let's get rid of the frontmatter drug array.

**Comment by dsifford** [comment]
Should the drug field be a custom markdown thing? It would be easier, more efficient, and more reliable to parse that out of an AST than it would to regex each card for html.

**Comment by dsifford** [comment]
@jvoros Is this issue still relevant? Didn't we pitch drugs?

**Comment by jvoros** [comment]
Yes. Drugs should be removed from cards. They are no longer in the build process.
